### ISSUE TITLE
Specify BeautifulSoup parser

### ISSUE THREAD

**Issue by jpadilla** [pyembed/pyembed#49]
To get rid of UserWarning.

```
/python2.7/site-packages/bs4/__init__.py:166: UserWarning: No parser was explicitly specified, so I'm using the best available HTML parser for this system ("lxml"). This usually isn't a problem, but if you run this code on another system, or in a different virtual environment, it may use a different parser and behave differently.
```

**Comment by jpadilla** [comment]
We'd just need to change https://github.com/pyembed/pyembed/blob/master/pyembed/core/parse.py#L73 to use `html.parser` which is the builtin parser:

```
soup = BeautifulSoup(oembed_response, 'html.parser')
```

**Comment by matt-thomson** [comment]
This is fixed in v1.2.1.  Thanks for reporting!

**Comment by matt-thomson** [comment]
Actually this happens in two places - you'll want to take v1.2.2.
